app/main/views/briefs.py

```diff
@@ -225,7 +225,7 @@ def redirect_to_next_page():
                     brief_response_id,
                     current_user.email_address
                 )
-
+                flash('submitted_first', 'success')
```

Comment:
`submitted_initial` sounds like it's an initial draft or something
`submitted_first_submit` sounds redundant
`first_submit` sounds like we're not sure if they'll have to submit again (at this point theyve been successful)
`submitted_success_first` ?